jdaviz/core/template_mixin.py

```diff
@@ -1255,14 +1255,22 @@ def _get_viewer(self, viewer):
         except TypeError:
             return self.app.get_viewer_by_id(viewer)
 
-    def _layer_to_dict(self, layer):
+    def _layer_to_dict(self, layer, label_to_color=None, label_mixed_color=None):
+        is_subset = False if hasattr(layer, 'state') and hasattr(layer.state, 'bitmap_visible') else True  # noqa
```

Comment:
This can be simplified to avoid the if/else (double check my logic below) but I also think we can just check the instance, right?

```suggestion
        is_subset = not hasattr(layer, 'state') or not hasattr(layer.state, 'bitmap_visible') else True  # noqa
```